### DISCUSSION TITLE
Bonito installation on Promethion

### DISCUSSION THREAD

**Discussion by RainerWaldmann** [nanoporetech/bonito#88] [Q&A]
Hi,

I would be interested to install Bonito on the Promethion sequencer. 
Does it run on the Promethion without any major  upgrade (Cuda ...) that might interfere with the sequencing process.

Rainer

**Comment by iiSeymour** [comment]
Hey @RainerWaldmann 

Currently, the PromethION has an older Nvidia driver and version of Cuda (9.0) that is required by Guppy so it's not recommended that you try and update these. Bonito can be made to work with Cuda 9.0 by installing an older version of PyTorch however it can be quite a bit slower due to less well-optimised software. Guppy will be able to run bonito models in the next release and we are working on a newer Ubuntu 20.04 base platform for PromethION that will bring all the latest drivers and SDKs.

HTH

Chris.
